azext_iot/tests/iothub/configurations/test_iot_config_int.py

```diff
@@ -96,6 +96,9 @@ def test_edge_deployments(self):
                 edge_content_malformed_path
             )
             self.kwargs["labels"] = '{"key0": "value0"}'
+            self.kwargs["custom_labels"] = "key0=value0"
```

Comment:
This entry should be different compared to the labels entry above, also `self.kwargs` is meant for inline json input  where during command execution the flow replaces the key, "{custom_labels}" with the escaped json payload.

Reply:
Hi Paymaun, now I updated the test to use self.kwargs for inline json for result checking, and input as kvp. is this what you were suggesting? thanks a lot!